commercetools.NET/DiscountCodes/DiscountCode.cs

```diff
@@ -80,6 +80,12 @@ public class DiscountCode
         [JsonProperty(PropertyName = "references")]
         public List<Reference> References { get; private set; }
 
+        /// <summary>
+        /// The custom fields.
+        /// </summary>
+        [JsonProperty(PropertyName = "custom")]
+        public CustomFields.CustomFields Custom { get; private set; }
```

Comment:
@jayS-de This is necessary because the class and namespace share the name CustomFields.  If you add the using statement and remove the namespace on line 87, you will get an error saying "'CustomFields' is a namespace but used like a type."